lib/portlayer/exec/container.go

```diff
@@ -517,8 +517,14 @@ func List(ctx context.Context, sess *session.Session, all *bool) ([]*Container,
 	if err != nil {
 		return nil, err
 	}
+
 	// convert to container
-	containers := convertInfraContainers(moVMs, all)
+	var t bool
+	if all != nil {
```

Comment:
Swagger represents optional vars as pointers so they can be set to `nil` if omitted.